### PR TITLE
[Admin QOL] Adminhelp flag permissions

### PR DESCRIPTION
**Pull request by Tainakov**
## About the PR

This PR gives QOL permissions to Adminhelp flag, at least i think it is. It adds permissions to filter ingame admin chat and use command "asay", so you can write in it.
## Why / Balance
Don't know where to start exactly. When you give Adminhelp flag to user, you can already see admin chat with his bwoink sounds, but you can't communicate in it, and you can't turn it off if needed.
Every moderator has access to Adminhelp (which is main tool to communicate with players), but not everyone who has access to Adminhelp is a moderator (We have mentor system on Corvax in our case)
So, since Adminhelp flag already gives acces to ingame admin chat, why not make it more useful? It hurts to communicate through ghost chat or Adminhelp.
## Technical details
I'm more than sure that there is a more elegant solution to this request, but i ended up with this. And it works fine.

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

### CODE REVIEW

File: Content.Client/UserInterface/Systems/Chat/ChatUIController.cs [hunk 1]
```diff
@@ -546,7 +546,7 @@ private void UpdateChannelPermissions()
         }
 
         // only admins can see / filter asay
-        if (_admin.HasFlag(AdminFlags.Admin))
+        if (_admin.HasFlag(AdminFlags.Adminhelp))
         {
             FilterableChannels |= ChatChannel.Admin;
             FilterableChannels |= ChatChannel.AdminAlert;
```
Comment (metalgearsloth) [inline Content.Client/UserInterface/Systems/Chat/ChatUIController.cs:552]: Codermins should be able to read admin chat but do not have ahelp perms.

Comment (Tainakov) [reply Content.Client/UserInterface/Systems/Chat/ChatUIController.cs]: Should be fine now, at least it doesn't break anything.

**Comment by moonheart08** [comment]
This should be done by introducing a new flag (+ADMINCHAT) and removing adminchat from other flags, not by spreading the permission out between flags.

**Comment by Tainakov** [comment]
> This should be done by introducing a new flag (+ADMINCHAT) and removing adminchat from other flags, not by spreading the permission out between flags.

Sounds reasonable and more clear. Thanks for idea, will come back with another PR when figure out how to create flags

**Pull request closed without merging**